test/core/mpi/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: MIT
+# SPDX-FileCopyrightText: 2023 NeoFOAM authors
+
+add_test(NAME test_operators COMMAND test_operators)
+add_executable(test_operators "test_operators.cpp" "../../test_main.cpp")
+target_link_libraries(
+  test_operators PRIVATE Catch2::Catch2 NeoFOAM Kokkos::kokkos
+                         cpptrace::cpptrace ${MPI_CXX_LIBRARIES})
```

unnecessary if the mpi target is linked publically.
```suggestion
                         cpptrace::cpptrace)
```